README.md

```diff
@@ -1,61 +1,81 @@
-# velocitas-cli
+Velocitas CLI - The project manager for your Vehicle Apps
+=================================
 
+[![CI Workflow](https://github.com/eclipse-velocitas/cli/actions/workflows/ci.yml/badge.svg#branch=main)](https://github.com/eclipse-velocitas/cli/actions/workflows/ci.yml)
 [![License: Apache](https://img.shields.io/badge/License-Apache-yellow.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-Velocitas CLI
-=================
+<!-- toc -->
+* [Introduction](#introduction)
+* [Problem](#problem)
+* [Solution](#solution)
+* [Getting started](#getting-started)
+* [Commands](#commands)
+* [Installation](#installation)
+* [Configuration](#configuration)
+* [Advanced topics](#advanced-topics)
+* [Contribution](#contribution)
+<!-- tocstop -->
+
+# Introduction
+
 The CLI implements the [lifecyle management](link-to-velocitas-docs) concept of the development environment of
```

Comment:
will probably be: https://websites.eclipseprojects.io/velocitas/docs/concepts/lifecycle_management/

Reply:
Done